config/k8s/wait.py

```diff
@@ -136,38 +142,47 @@ def main(args):
     if pod is None:
         print("ERROR: Failed to find pod")
         sys.exit(1)
-    if len(pod.items) < 1:
```

Comment:
FYI this whole `wait.py` thing is going away with the new API as the jobs will be sending their results to the API directly and that will generate an event so nothing will be waiting.  While it's nice to fix things, I would say that if it breaks less than 5% of the time it's not worth the effort.  That time could be much better spent on ramping up the new API instead.

Reply:
I just took a few minutes to finalize this PR(as its hanging for quite long time due low priority), as I've observed this is all that makes sense to improve at the moment.